### PR TITLE
fix(workspace): make analytics icon visibility configurable

### PR DESCRIPTION
**Pull request by thnkloud9**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -219,6 +219,14 @@ function getDefaults(grunt) {
             useTansaProofing: false
         },
 
+        // workspace defaults
+        workspace: {
+            ingest: false,
+            content: false,
+            tasks: false,
+            analytics: true
```
Comment (petrjasek) [inline webpack.config.js]: imo this should b `false` now, otherwise we might need to modify all instances

Comment (thnkloud9) [inline webpack.config.js]: ok by me, but then should we set it to true in the superdesk repo?

Comment (petrjasek) [inline webpack.config.js]: yep we could enable it on master

Comment (thnkloud9) [inline webpack.config.js]: cool

**Pull request merged**